tests/o3/irreps_test.py

```diff
@@ -19,6 +19,8 @@ def test_creation():
     o3.Irreps([(16, "3e"), '1e'])
     o3.Irreps([(16, "3e"), '1e', (256, (1, -1))])
 
+    assert o3.irreps.l3e == ir
```

Comment:
Is it `o3.irreps` or `o3.irrep` ? `o3.irrep` looks better to me

Reply:
irrep it is!